message_sender/tasks.py

```diff
@@ -77,6 +80,71 @@ def run(self, metric_name, metric_value, session=None, **kwargs):
 fire_metric = FireMetric()
 
 
+class ConcurrencyLimiter(object):
+    BUCKET_SIZE = 60
+
+    @classmethod
+    def get_current_message_count(cls, msg_type, delay):
+        # Sum the values in all the buckets to get the total
+        total = 0
+        number_of_buckets = delay // cls.BUCKET_SIZE + 1
+        bucket = int(time.time() // cls.BUCKET_SIZE)
+        for i in range(0, number_of_buckets):
+            value = cache.get(msg_type+"_messages_at_%s" % bucket)
```

Comment:
We're mixing string concatenation and string formatting here. Could we rather do `"%s_messages_at_%s" % (msg_type, bucket)`, or use `.format`?